### PR TITLE
Ohie 395 use labels for identifying objects in a package

### PR DESCRIPTION
**Pull request by bradsawadye**
Labels are now used for identifying components of a package. Namespaces were being used for this but this has been changed as the two packages (the core and healthworkforce) share a database. This will ensure that the database is easily accessible to the components

### CODE REVIEW

File: healthworkforce/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Kubernetes
+
+Navigate to the folder `/kubernetes/main` to execute the commands.
+
+To set up the healthworkforce services run the following command:
+
+```sh
+./k8s.sh init
+```
```
Comment (BMartinos) [inline healthworkforce/README.md:11]: I dont think there is an init script for the hfw package? this was only needed in the core package?

File: healthworkforce/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Docker
+
+Navigate to the folder `/Docker` to execute the commands.
+
+To set up the healthworkforce services run the following command:
+
+```sh
+./compose.sh init
+```
```
Comment (BMartinos) [inline healthworkforce/README.md:39]: Same as above comment, I dont we have an init command for the hwf package

File: healthworkforce/kubernetes/importer/volume/openhim-import.json [hunk 3]
```diff
@@ -48,18 +53,18 @@
       "requestBody": true,
       "responseBody": true,
       "rewriteUrlsConfig": [],
-      "name": "Mapping mediator",
+      "name": "mCSD mediator",
+      "description": "Triggers the retrieving of data and the mapping of that data into a FHIR bundle, which will then be sent to a FHIR metadata store ",
+      "pollingSchedule": "0 * * * *",
       "urlPattern": "^.*$",
```
Comment (BMartinos) [inline healthworkforce/kubernetes/importer/volume/openhim-import.json:59]: Lets update this patern to not be a `catch all`. Anything request that doesnt meet the other patterns will end up triggering this polling channel
```suggestion
      "urlPattern": "^/mcsd-trigger$",
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,67 @@
...
+
+Navigate to the main folder to execute the commands.
+
+To set the Instant OpenHIE services run the following command:
+
+```sh
+./instant.sh init k8s
+```
+
+To tear down the deployments use the opposing command:
```
Comment (BMartinos) [inline README.md:51]:
```suggestion
To tear down the deployments use the following command:
```

File: healthworkforce/README.md (hunk 1 above)
Comment (bradsawadye) [reply healthworkforce/README.md]: addressed in https://github.com/openhie/instant/pull/45/commits/ac4a7df3cba21c52d89937795d33e9cc349da517

File: healthworkforce/README.md (hunk 2 above)
Comment (bradsawadye) [reply healthworkforce/README.md:45]: We have an init command for the docker stuff. Its where we do the setting of the mscd endpoint

**Pull request merged**